doc/classes/NavigationMeshInstance.xml

```diff
@@ -1,23 +1,41 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NavigationMeshInstance" inherits="Spatial" version="3.5">
 	<brief_description>
-		Node that instances navigation meshes into a scenario.
+		An instance of a [NavigationMesh].
 	</brief_description>
 	<description>
-		NavigationMeshInstance is a node that takes a [NavigationMesh] resource and adds it to the current scenario by creating an instance of it.
+		An instance of a [NavigationMesh]. It tells the [Navigation] node what can be navigated and what cannot, based on the [NavigationMesh] resource. This should be a child of a [Navigation] node.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="bake_navigation_mesh">
+			<return type="void" />
+			<description>
+				Bakes the [NavigationMesh]. The baking is done in a seperate thread because navigation baking is not a cheap operation. This can be done at runtime. When it is completed, it automatically sets the new [NavigationMesh].
+			</description>
+		</method>
 	</methods>
 	<members>
-		<member name="enabled" type="bool" setter="set_enabled" getter="is_enabled" default="true">
-			If [code]true[/code], the navigation mesh will be used by [Navigation].
+		<member name="enabled" type="bool" setter="set_enabled" getter="is_enabled" default="false">
```

Comment:
I noticed enabled is false by default now. What's the reason for that? I think it's going to break backwards compatibility.

Reply:
This was probably just an oversight for the original implementation.